demo/style.css

```diff
@@ -14,6 +14,7 @@ body {
 
 input:focus,
 a:focus,
-button:focus {
+button:focus,
+.highlighted {
```

Comment:
I think it's confusing that this looks exactly the same as focus. Let's give it a different appearance.